multiqc/modules/preseq/preseq.py

```diff
@@ -95,22 +193,42 @@ def preseq_length_trimmed_plot (self):
             'ymin': 0,
             'xmin': 0,
             'tt_label': '<b>{point.x:,.0f} total</b>: {point.y:,.0f} unique',
```

Comment:
Here's the thing with the ints: `{point.x:,.0f}` (not usually a problem for counts, but annoying when coverage). Might be nice to add the suffix to the tooltip too (`molecules` or `X coverage`).